src/System.Devices.Gpio/System.Devices.Gpio/GPIOPin.cs

```diff
@@ -6,8 +6,10 @@ namespace System.Devices.Gpio
 {
     public enum GpioPinMode
     {
-        Input = 0,
-        Output = 1
+        Input,
```

Comment:
one more thing, can this be a flag instead such that writing "LightSensor.PinMode = GPIOPinMode.PullUp | GPIOPinMode.Input;" is possible?

Reply:
Yes, Input configures the pin to be just input with pull up/down disabled.

I thought about making those constants flags but since pull up/down options only make sense when they are combined with Input I decided that is more clear to do it without flags. That's also why I prefixed with Input both pull up/down constants.

Also, later on I will add PWM option (as well as SPI, I2C and other more advanced pin modes) and I thought it will be easier to understand if I prefix that constant name with Output, like OutputPWM. Other wise the user will need to know which flags can be combined with each other to generate valid modes. For example, GPIOPinMode.PullUp | GPIOPinMode.Output doesn't make any sense (I believe), nor GPIOPinMode.PWM | GPIOPinMode.Input.